Podfile

```diff
@@ -85,6 +85,73 @@ def shared_with_extension_pods
     pod 'Down', '~> 0.6.6'
 end
 
+def shared_with_app_target_pods
```

Comment:
This approach works, but is IMHO somewhat fragile – as a for-instance, if you need to customize WPAuth for Jetpack, how would you specify a different version for each?

The better approach here is probably an `abstract_target` (see https://guides.cocoapods.org/using/the-podfile.html and search for "abstract target"). This allows inheritance, which is probably preferable to composability in this case IMHO.

Reply:
Good point! Thanks for the tip. Fixed in a905e59